streamx-console/streamx-console-service/src/main/java/com/streamxhub/streamx/console/core/service/impl/AlertServiceImpl.java

```diff
@@ -37,21 +42,22 @@
 import org.springframework.stereotype.Service;
 
 import javax.annotation.PostConstruct;
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
 import java.io.File;
 import java.io.StringWriter;
 import java.net.URL;
-import java.util.Date;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.TimeZone;
+import java.net.URLEncoder;
+import java.util.*;
```

Comment:
the name of the referenced package needs to be clearly defined, otherwise it will not pass the checkstyle check